app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -163,10 +186,22 @@ class OfferViewModelImpl(
     override fun approveOffer() {
         viewModelScope.launch {
             _viewState.value = _viewState.value.copy(isLoading = true)
+            val postSignDependencies = getPostSignDependenciesUseCase(quoteIds)
```

Comment:
When using `invoke` functions without explicitly calling `.invoke()` it makes it impossible for AS to find the usage when doing Cmd + B on the `invoke()` function itself.
It does work on latest IntelliJ version so it seems like an AS bug, but we all know these can take many months before they are integrated to AS too.
I would be happy if we considered explicitly calling `invoke()` on the UseCase functions as it doesn't make it any less readable or whatever, and it really helps with navigating the code (finding usages of the useCase in one go instead of Searching for the name of the useCase as a string)

But wouldn't protest if you don't like it, it's just a mild inconvenience

Reply:
Either style is fine by me. Maybe you can weigh in, @hugokallstrom?

Reply:
Explicitly calling invoke() makes sense tbh. Its not as pretty but worth it being able to navigate via Cmb + B.

Reply:
```suggestion
            val postSignDependencies = getPostSignDependenciesUseCase.invoke(quoteIds)
```

Reply:
`invoke` it is.